util/dex/dex.go

```diff
@@ -83,7 +84,10 @@ func NewDexHTTPReverseProxy(serverAddr string, baseHRef string, tlsConfig *DexTL
 			if err != nil {
 				return err
 			}
-			log.Errorf("received error from dex: %s", string(b))
+			log.WithFields(log.Fields{
+				common.SecurityField:    common.SecurityMedium,
+				common.SecurityCWEField: 775,
```

Comment:
@my-git9 can you drop this line? CWE-775 is specific to failed file closing. I'm not sure there's a specific CWE that makes sense for an error message from Dex.